website/docs/reference/rbac.md

```diff
@@ -192,7 +192,7 @@ You can assign the following permissions on a per-environment level within the p
 | **update variants**              | Lets the user create, edit and remove variants within the environment.                                                                                                                                                                                                                                                                                |
 | **approve a change request**     | Lets the user approve [change requests](./change-requests) in the environment.                                                                                                                                                                                                                                                                       |
 | **apply a change request**       | Lets the user apply change requests in the environment.                                                                                                                                                                                                                                                                                               |
-| **skip change requests**         | Lets the user ignore change request requirements. This applies **only when using the API** directly; when using the admin UI, users with this permission will still need to go through the normal change request flow. You can find more details in the section on [circumventing change requests](./change-requests#circumventing-change-requests). |
+| **skip change requests**         | Lets the user skip the change request process for a project and environment where change requests are enabled.  |
```

Comment:
When I tested this using the UI, I was able to skip the change request there too. Am I missing something?

Reply:
Hmmm, good question. I'd probably check with @kwasniew for this. Maybe we changed it? If not, it might be a bug 🐛